recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
```

Comment:
IIRC, when I last built `libhdfs3` by hand, I recall it not working with newer versions of boost. This might have changed in the latest minor release, but I haven't tried.